kld/src/cli/main.rs

```diff
@@ -179,6 +179,14 @@ enum Command {
     },
     /// Fetch the aggregate local and remote channel balances (msat) of the node
     LocalRemoteBalance,
+    /// Get node routing fees.
+    GetFees,
+    /// Fetch a list of the forwarded htlcs.
+    ListForwards {
+        /// The status of the forwards (settled, failed)
+        #[arg(long)]
+        status: Option<String>,
```

Comment:
Could we use `enum` for these? Or reuse the `ForwardStatus` in `kld/src/database/forward.rs`

```rust
enum Status {
  Settled,
  Failed
}

use std::str::FromStr;
type ParseError = &'static str;

impl FromStr for Status {
    type Err = ParseError;
    fn from_str(status: &str) -> Result<Self, Self::Err> {
        match status {
            "settled" => Ok(Status::Settled),
            "failed" => Ok(Status::Failed),
            _ => Err("Only status of the forwards allow now (settled,failed)"),
        }
    }
}
```

Reply:
The type it should be is GetV1ChannelListForwardsResponseItemStatus. But this is generated and doesn't seem to work well with clap due to missing traits. So I think we just have to leave as it is for now.

Reply:
Just wondering, if the automatically generated code does not work, why not we write one and fix it?

Reply:
It's just less important to me than most other work we have to do. The API does the validation. CLI is a dumb client.